### PR TITLE
Updates after organization rename

### PR DESCRIPTION
**Pull request by pkliczewski**
This PR removes usage of old name (Parodos) and fixes links.

### CODE REVIEW

File: .github/workflows/m2k-func.yaml [hunk 1]
```diff
@@ -99,7 +99,7 @@ jobs:
             ${PR_OR_COMMIT_URL} \
             "yq --inplace '.kfunction.image=\"quay.io/orchestrator/serverless-workflow-m2k-kfunc:${GITHUB_SHA}\"' charts/move2kube/values.yaml" \
             scripts/create_automated_m2k_kfunc_pr.sh \
-            ${{ github.event.head_commit.author.email || 'parodos@redhat.com' }} \
+            ${{ github.event.head_commit.author.email || 'rhdhorchestrator@redhat.com' }} \
```
Comment (masayag) [inline .github/workflows/m2k-func.yaml:102]: @pkliczewski do we have that email address set?

Comment (pkliczewski) [reply .github/workflows/m2k-func.yaml:102]: not yet

**Comment by masayag** [comment]
/lgtm
@gabriel-farache is there okay from konflux perspective?

**Comment by gabriel-farache** [comment]
@pkliczewski could you reopen a PR on the upstream itself and not from your own fork please? The CIs are using secrets which can only be used when PR opened from upstream branch

**Comment by pkliczewski** [comment]
In order to run the CI with our secrets configured I created a [PR](https://github.com/rhdhorchestrator/serverless-workflows/pull/551) from a branch rather than a fork.

**Pull request closed without merging**